hr_timesheet_task_domain/tests/test_hr_timesheet_task_domain.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016-2018 Tecnativa - Pedro M. Baeza
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import TransactionCase
+
+
+class TestHrTimesheetTaskDomain(TransactionCase):
+    def setUp(self):
+        super().setUp()
+
+        self.project = self.env["project.project"].create({"name": "Test project"})
+        self.analytic_account = self.project.analytic_account_id
+        self.task = self.env["project.task"].create(
+            {"name": "Test task", "project_id": self.project.id}
+        )
+        self.line = self.env["account.analytic.line"].create(
+            {
+                "task_id": self.task.id,
+                "account_id": self.analytic_account.id,
+                "name": "Test line",
```

Comment:
```suggestion
                "name": "Test line",
                "employee_id": 2,
```

We need an employee when we create a timesheet. This corrects the tests.

Reply:
@Yadier-Tecnativa complete this